test/lish/does_not_include_dot_file.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+@Timeout(Duration(minutes: 30))
+
+import 'dart:convert';
+
+import 'package:shelf/shelf.dart' as shelf;
+import 'package:test/test.dart';
+
+import 'package:pub/src/exit_codes.dart' as exit_codes;
+
+import '../descriptor.dart' as d;
+import '../test_pub.dart';
+import 'utils.dart';
+
+void main() {
+  setUp(d.validPackageWithDotFiles.create);
```

Comment:
Since `validPackageWithDotFiles` is only used in this file, can we maybe just move it into this file.